spec/rex/parser/arguments_spec.rb

```diff
@@ -50,14 +52,72 @@
 
       it 'ignores combined flags that do not exist' do
         input = ['-unknown-flags']
```

Comment:
With the above comment, I think I understand this a bit better now

Is this test different now, because of the regex addition which no longer supports`-` in the middle of short flags, so it's not testing how unknown flags are handled now? If that's the case this test will need to be updated to still test what was previously being tested

Reply:
If I'm understanding correctly, then this should be addressed in the latest commit 👍 
Correct me if I'm wrong/you'd still like this addressed differently.